src/otx/core/model/module/classification.py

```diff
@@ -39,32 +40,31 @@ def __init__(
         torch_compile: bool,
         optimizer: list[OptimizerCallable] | OptimizerCallable = lambda p: torch.optim.SGD(p, lr=0.01),
         scheduler: list[LRSchedulerCallable] | LRSchedulerCallable = torch.optim.lr_scheduler.ConstantLR,
+        metric: Metric = Accuracy,
```

Comment:
In my opinion, `OTXLitModule` should also accept a metric argument.
Also, common code like `metric.reset` should probably go in an `OTXLitModule`.